services/gateway/apis/http-version3/methods/transactionsStatistics.js

```diff
@@ -44,15 +44,21 @@ module.exports = {
 					$ref: '#/definitions/TransactionsStatisticsWithEnvelope',
 				},
 			},
+			503: {
+				description: 'Service Unavailable',
+				schema: {
+					$ref: '#/definitions/serviceUnavailable',
+				},
+			},
 		};
-		transactionSchema[this.swaggerApiPath].get.parameters = [{
+		transactionSchema[this.swaggerApiPath].get.parameters[0] = {
```

Comment:
It works correctly. We are overriding only the first parameter i.e. `interval` rather than all parameters which resulted in removal of limit and offset.